src/mono/cmake/config.h.in

```diff
@@ -517,9 +517,6 @@
 /* Have getifaddrs */
 #cmakedefine HAVE_GETIFADDRS 1
```

Comment:
@simonrozsival we can also remove this one, it is unused in mono now

Reply:
@simonrozsival that is part of System.Native, not mono. It has its own check: https://github.com/dotnet/runtime/blob/f249dc77c6614e05fe47ac9f5eae987e0ab12036/src/native/libs/configure.cmake#L139-L141